examples/molecular_dynamics/example_molecular_dynamics.cpp

```diff
@@ -94,7 +94,7 @@ int main(int argc, char *argv[])
         particles[4 * id + 3] = {i * dx + .5f, j * dy + .5f, k * dz + .0f};
       });
 
-  Kokkos::View<float * [3], MemorySpace> velocities(
+  Kokkos::View<float *[3], MemorySpace> velocities(
```

Comment:
I like that new formatting of view data type with mixed dynamic and static dimensions